content/kubeflow/install.md

```diff
@@ -5,31 +5,37 @@ weight: 10
 draft: false
 ---
 
-In this chapter, we will install Kubeflow on Amazon EKS cluster. The cluster creation steps are outlined in [Launch EKS](/eksctl/launcheks/) chapter.
+In this chapter, we will install Kubeflow on Amazon EKS cluster. If you don't have an EKS cluster, please follow instructions from [getting started guide] (/prerequisites) and then launch your EKS cluster using [eksctl](/eksctl) chapter
 
 ### Install Kubeflow on Amazon EKS
 
-Download 0.6.1+ release of `kfctl`. This binary will allow you to install Kubeflow on Amazon EKS:
+Download 0.7 RC6 release of `kfctl`. This binary will allow you to install Kubeflow on Amazon EKS:
```

Comment:
RC6 could be outdated this weekend. Could we ask user use latest? Or do we want to maintain the changes?

Reply:
I think we should keep specific versions that work rather than using latest. I'll keep an eye and submit changes